src/components/PollSection/usePoll.ts

```diff
@@ -9,7 +9,10 @@ export const usePoll = (pollId: string, shouldFetch: boolean) => {
     if (pollData?.closedAt) {
       const endDate = dayjs(pollData.closedAt);
       const daysRemaining = endDate.diff(dayjs(), 'day');
-      return daysRemaining || 0;
+      if (daysRemaining > 0) return daysRemaining;
+      const hoursRemaining = endDate.diff(dayjs(), 'hour');
+      if (hoursRemaining > 0) return 1;
+      return 0;
```

Comment:
Why not just return `hoursRemaining`

```suggestion
      if (daysRemaining > 0) return daysRemaining;
      const hoursRemaining = endDate.diff(dayjs(), 'hour');
      return hoursRemaining > 0 ? hoursRemaining : 0
```

Reply:
UI is showing only days in figma xD

Reply:
it means if `hoursRemaining = 2` hr in UI will show as 1 day, am I correct ?

Reply:
yes. exactly.